targets/python/make.js

```diff
@@ -0,0 +1,407 @@
+        parents.push("PlayFabHTTP.PlayFabResultCommon");
+    else {
+        parents.push("PlayFabHTTP.PlayFabBaseObject");
+    }
+
+    //parents.push("PlayFabHTTP.Serializable")
+
+    if (parents.length > 0) {
+        var output = "(";
+        for (var i = 0; i < parents.length; i++) {
+            if (i !== 0)
+                output += ", ";
+            output += parents[i];
+        }
+        output += ")"
+    }
+    return output;
+}
+
+function getPropertyAttribs(tabbing, property, datatype, api) {
+    var attribs = "";
+
+    if (property.isUnordered) {
+        var listDatatype = api.datatypes[property.actualtype];
+        if (listDatatype && listDatatype.sortKey)
+            attribs += tabbing + "# [Unordered(SortProperty=\"" + listDatatype.sortKey + "\")]\n";
+        else
+            attribs += tabbing + "# [Unordered]\n";
+    }
+
+    return attribs;
+}
+
+function getModelPropertyDef(property, datatype) {
+    var basicType;
+    if (property.collection) {
+        //basicType = getPropertyCsType(property, datatype, false);
+
+        //if (property.collection === "array")
+        //    return "List<" + basicType + "> " + property.name;
+        //else if (property.collection === "map")
+        //    return "Dictionary<string," + basicType + "> " + property.name;
+        //else
+        //    throw "Unknown collection type: " + property.collection + " for " + property.name + " in " + datatype.name;
+    }
+    else {
+        //basicType = getPropertyCsType(property, datatype, true);
+        //return basicType + " " + property.name;
+    }
+    return property.name;
+}
+
+function getAuthParams(apiCall) {
+    if (apiCall.url === "/Authentication/GetEntityToken")
+        return "authKey, authValue";
+    if (apiCall.auth === "EntityToken")
+        return "\"X-EntityToken\", EntityToken";
+    if (apiCall.auth === "SecretKey")
+        return "\"X-SecretKey\", DeveloperSecretKey";
+    else if (apiCall.auth === "SessionTicket")
+        return "\"X-Authorization\", PlayFabSettings.ClientSessionTicket";
+    return "None, None";
+}
+
+function getRequestActions(tabbing, apiCall) {
+    if (apiCall.result === "LoginResult" || apiCall.request === "RegisterPlayFabUserRequest")
+        return tabbing + "request[\"TitleId\"] = PlayFabSettings.TitleId or request.TitleId\n"
+            + tabbing + "if request[\"TitleId\"] == None:\n"+tabbing+"    raise Exception(PlayFabExceptionCode.TitleNotSet, \"Must be have TitleId set to call this method\")\n";
+    if (apiCall.auth === "EntityToken")
+        return tabbing + "if PlayFabSettings.EntityToken == None:\n "+tabbing+"    raise Exception(PlayFabExceptionCode.EntityTokenNotSet, \"Must call GetEntityToken before calling this method\")\n";
+    if (apiCall.auth === "SessionTicket")
+        return tabbing + "if PlayFabSettings.ClientSessionTicket == None:\n"+tabbing+"    raise Exception(PlayFabExceptionCode.NotLoggedIn, \"Must be logged in to call this method\")\n";
```

Comment:
Replace:
if PlayFabSettings.ClientSessionTicket == None:
if not PlayFabSettings.ClientSessionTicket:

Reply:
Exception -> PlayFabException

Reply:
(and similarly in this whole section)